LuisPalominoTrevilla/TaskManagerSystem
# Service registry

## Prerequisites

- Have an .env file. (Ask me for it)

## Test

- Start the service registry
- Start the accounts microservice
- The microservice should connect to the service registry
- You can now create requests to localhost:3000/api/v1/accounts
